addons/account/models/account.py

```diff
@@ -100,7 +100,7 @@ def _check_reconcile(self):
         help="Forces all moves for this account to have this account currency.")
     code = fields.Char(size=64, required=True, index=True)
     deprecated = fields.Boolean(index=True, default=False)
-    user_type_id = fields.Many2one('account.account.type', string='Type', required=True, oldname="user_type", 
+    user_type_id = fields.Many2one('account.account.type', string='Type', required=True, oldname="user_type",
```

Comment:
Try to not make PR with unecessary changes those will be rejected by default, or at least you will increase the work of the merger.

Reply:
Of course, you are right. This case is somewhat special: Its an automated process which does some basic auto formatting, which I commonly use. Its not explicitly intentional...